edge-modules/load-gen/Settings.cs

```diff
@@ -61,6 +59,8 @@ public class Settings
 
         public string OutputName { get; }
 
+        public int StartDelay { get; }
+
         public override string ToString()
         {
             return JsonConvert.SerializeObject(this, Formatting.Indented);
```

Comment:
nit: Philip wants to move away from the Json serialization pattern as it makes it easy to accidentally leak secrets.

Reply:
Changed it and added comment explaining why